static/js/main.js

```diff
@@ -38,56 +38,64 @@ const createInputs = () => {
     {
       name: SolenoidParameters.VOLTAGE,
       symbol: 'V',
-      value: urlParams.get(SolenoidParameters.VOLTAGE) || '',
+      value: urlParams.get('voltage') || '',
```

Comment:
Just curious, is there a reason you decided to use the literal strings instead? Everything else LGTM

Reply:
Nah, just thought it was cleaner